README.md

```diff
@@ -61,8 +63,17 @@ None.
       - role: dj-wasabi.telegraf
 ```
 
-- For deploying producer script and set up cron job to run it.
-_Note: Make sure to update the `pulsar_app_dir` variable's value._
+### Deployment of the producer
+
+Please pay special attention to the value of these two variables
+in the [defaults/main.yml](defaults/main.yml) file
+* `pulsar_app_dir`
+* `pulsar_root`
```

Comment:
isn't it supposed to be `pulsar_data_path`? That's what is being modified in the playbook below, right?

Reply:
you are right, sorry.

actually, on second thoughts, should we instead specify `pulsar_root` and `pulsar_data_path`? since there are other variables dependent on them

Reply:
You are right; the two should be enough as the others are already using it. 

```
pulsar_data_path
pulsar_root
```

Reply:
Great thanks! 

Will you approve or should we wait for @pauldg to review as well?

Reply:
I can approve and wait until tomorrow to merge it if @pauldg has some feedback.